README.md

```diff
@@ -50,6 +50,15 @@ The response correspond to the following schema:
 
 ## Tests
 
+In order to run tests locally you also need to pull submodules. The easiest way to do that is cloning repo with
+` --recursive` flag:
+
+```
+  git clone --recursive git@github.com:Runnable/octobear.git
+```
+
+Also, in order to run tests locally you need to have `configs/.env.test` file.
```

Comment:
> Also, in order to run tests locally you need to have `configs/.env.test` file.

Also, in order to run tests locally you'll need populate the environment variables in `configs/.env`. We suggest adding them to `configs/.env.test`.